api/v1alpha1/virtualmachineclass_types.go

```diff
@@ -80,6 +80,8 @@ type VirtualMachineClassStatus struct {
 // +kubebuilder:printcolumn:name="CPU",type="string",JSONPath=".spec.hardware.cpus"
 // +kubebuilder:printcolumn:name="Memory",type="string",JSONPath=".spec.hardware.memory"
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
+// +kubebuilder:printcolumn:name="VGPUDevices",type="string",priority=1,JSONPath=".spec.hardware.devices.vgpuDevices"
```

Comment:
Suggestion for doing `.spec.hardware.devices.vgpuDevices[*].profileName ` would result in just the profile names (without the curly braces and field names)

Reply:
This makes sense to me, I will rename the column name to `VGPUDevicesProfileNames` and just list the profile names as values